sample_metadata/parser/generic_metadata_parser.py

```diff
@@ -415,12 +415,11 @@ def prepare_dict_from_keys(key_parts: List[str], val):
 
         return reduce(GenericMetadataParser.merge_dicts, dicts)
 
-    async def get_read_filenames(self, sample_id: str, row: GroupedRow) -> List[str]:
+    async def get_read_filenames(self, sample_id: str, row: SingleRow) -> List[str]:
```

Comment:
Was the GroupedRow / SingleRow definition wrong before hand? My thought would have been a GroupedRow, but as long as tests are passing it doesn't really matter